### PR TITLE
Added 3 questions to Languages -> Android -> questions

### PR DESCRIPTION
**Pull request by Sparsh011**
## What type of PR is this? (check all applicable)

- [ ] 🚀 Added Name
- [ ] ✨ Feature
- [ ] 🌟 stared the repo
- [ ] 🐛 Grammatical Error
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

### CODE REVIEW

File: Languages/Android/questions.json [hunk 1]
```diff
@@ -550,6 +550,33 @@
...
       "shortAns": "It attaches and detaches fragments from their host activity.",
       "tag": "fragment"
+    },
+    {
+      "id": 61,
+      "question": "To pass an object to different activities, which interface should the object implement?",
+      "details": "To pass an object to different activities, the object should implement either Serializable or Parcelable interface.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "It should implement either serializable or parcelable.",
```
Comment (Tc2r1) [inline Languages/Android/questions.json:560]: This question and shortAns are way too obvious. Please phrase the shortAns as "It is ..." and please dont use the wording of the question itself in the answer, that is a dead give away.

File: Languages/Android/questions.json [hunk 2]
```diff
@@ -550,6 +550,33 @@
...
+      "tag": "intent"
+    },
+    {
+      "id": 62,
+      "question": "What is a dangerous permission?",
+      "details": "Permissions that are dangerous include those that affect user data in some way. For example reading or writing data from device's storage",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "It is a permission that may affect user's privacy.",
```
Comment (Tc2r1) [inline Languages/Android/questions.json:569]: Again please don't use the word in the shortAns. 

"what does a cat eat?" 
a) oranges
b) tacos
c) a cat eats mice
d) trees. 

making the answer so obvious defeats the point of the app.

File: Languages/Android/questions.json [hunk 3]
```diff
@@ -550,6 +550,33 @@
...
+      "tag": "permissions"
+    },
+    {
+      "id": 63,
+      "question": "What is a pending intent?",
+      "details": "A PendingIntent object wraps the functionality of an Intent object while allowing your app to specify something that another app should do, on your app’s behalf, in response to a future action. For example, the wrapped intent might be invoked when an alarm goes off, or when the user taps on a notification.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "It allows foreign applications to execute our app's code",
```
Comment (Tc2r1) [inline Languages/Android/questions.json:578]: This one is good.

**Review by Tc2r1 — CHANGES_REQUESTED** [review]

File: Languages/Android/questions.json (hunk 3 above)
Comment (Sparsh011) [reply Languages/Android/questions.json:578]: Okay, I'll make the necessary changes

**Comment by Sparsh011** [comment]
I have changed the questions, please review them.

**Comment by Tc2r1** [comment]
Please dont' 4get to star the repository

**Review by AndroidLion48 — APPROVED** [review]
Thank you. Well done!

**Pull request merged**